3.3/Dockerfile

```diff
@@ -55,16 +59,22 @@ RUN wget -O redmine.tar.gz "https://www.redmine.org/releases/redmine-${REDMINE_V
 
 RUN set -eux; \
 	\
+	apt-mark auto \
+		dirmngr \
+		gnupg \
+	; \
```

Comment:
These lines won't save any space anyway, they should just be dropped. If we want to be able to remove them cleanly they would have to be installed and removed in the layer that they are used.

Reply:
Done

Reply:
I filled #122 for the testing issue.